packages/manager/modules/pci/src/projects/project/data-processing/data-processing.constants.js

```diff
@@ -95,4 +95,8 @@ export const GIB_IN_MIB = 1024;
 export const NOTEBOOK_CLUSTER_NAME = 'NB1-1';
 
 export const DATA_PROCESSING_TRACKING_PREFIX =
-  'PublicCloud::pci::projects::project::data-processing';
+  'pci::projects::project::data-processing';
+
+export const PUBLIC_CLOUD_TRACKING_PREFIX = 'PublicCloud';
+
+export const DATA_PROCESSING_TRACKING_PREFIX_FULL = `${PUBLIC_CLOUD_TRACKING_PREFIX}::${DATA_PROCESSING_TRACKING_PREFIX}`;
```

Comment:
**nitpick:** I noticed that `DATA_PROCESSING_TRACKING_PREFIX` and `DATA_PROCESSING_TRACKING_PREFIX_FULL` are maintained which can be confusing. 
Can you please create a refactoring task to change this implementation and maintain only one constant? So that we dont block the prod of this PR at the last moment.
P.S: Sorry that I didn't bring this during the first review.